src/pages/[community_id].js

```diff
@@ -0,0 +1,167 @@
+import Link from "next/link";
+import NavBar from "@/components/Header/NavBar";
+import CustomizableAvatar from "@/components/common/customavatar/CustomizableAvatar";
+import CustomizableButton from "@/components/common/CustomizableButton";
+import FooterBar from "@/components/common/FooterBar";
+import styles from "@/styles/singleListing.module.css";
+import Image from "next/image";
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import { getDocs, collection } from "firebase/firestore";
+import { firebaseDb } from "@/firebase/config";
+
+function SingleListing(className, text) {
+  const photo = "/community dashoard  img 1.jpg";
+  const Manager_Avatar = "/avarar 1.png";
+  const AVATAR_WIDTH = 50;
+  const AVATAR_HEIGHT = 50;
+  const SmallAvatar_WIDTH = 20;
+  const SmallAvatar_HEIGHT = 20;
+  const LOGO_WIDTH = 20;
+  const LOGO_HEIGHT = 20;
+
+  const router = useRouter();
+  const id = router?.query?.community_id;
+  const title = router?.query?.title;
+  const [listing, setListing] = useState();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const querySnapshot = await getDocs(
+        collection(firebaseDb, "MANAGERS", title, "MANAGER_LISTINGS")
+      );
+
+      querySnapshot.forEach((doc) => {
+        if (doc.id === id) {
+          setListing(doc.data());
+        }
+      });
+    };
+    if (id) {
+      fetchData();
+    }
+  }, [id, listing, title]);
+
+  return (
+    <>
+      <NavBar />
+      <header className="">
```

Comment:
This file is getting on the large side. I would move this header into its own component file and import it here. WIll make it easier to read and a header is a natural abstraction.

Reply:
Ok. I will do that

Reply:
@jjleigh I have created a header component and imported back to the community page

Reply:
@jjleigh I have created a header component and imported back to the community page